src/main/java/seedu/address/model/task/Task.java

```diff
@@ -12,6 +12,7 @@ public class Task {
 
     // identity fields
     private final Title title;
+    private boolean marked;
```

Comment:
This field should be final and Task should be immutable! When marking a task as done, can create a new Task object instead.

Reply:
Following this comment, will it be better if there was no "markTask" method in Model and ModelManager? Instead will it be better if we follow the add remark feature tutorial and use model.setPerson(personToEdit, editedPerson) in markTaskCommand::execute?

Reply:
I understand, but not sure about model.setTask. Maybe you can help me with that first.

Reply:
Noted! I will make change on this part. 👍

Reply:
Finished.